docs/pages/debugging/create-devtools-plugins.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Create a dev tools plugin
+description: Learn how to create a dev tools plugin to enhance your development experience.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+
+> **warning** Dev Tools plugins are compatible with SDK 50 and above. The process for creating a plugin is still in beta and may change in the future.
+
+You create a dev tools plugin, whether that's for inspecting aspects of a common framework or library or something specific to your custom code.
```

Comment:
```suggestion
You can create a dev tools plugin, whether that's for inspecting aspects of a common framework or library or something specific to your custom code. This guide will walk you through creating a dev tools plugin.
```

Add the context as to what this guide does for me as a reader (and why I should care about it).